src/Backend/Core/Layout/Templates/macros.html.twig

```diff
@@ -36,19 +36,14 @@
   <abbr tabindex="0" data-toggle="tooltip" aria-label="{{ title }}" title="{{ title }}">{{ macro.icon('info-circle') }}</abbr>
 {% endmacro %}
 
-{% macro dataGrid(dataGrid, noItemsMessage) %}
+{% macro dataGrid(dataGrid, addItemLink, addItemText, noItemsText) %}
```

Comment:
Isn't it more logical to put the `noItemsText` first, this will always be available, the `addItemsLink` and `addItemsText` might not always be available

Reply:
No it is not in my eyes, because the last two items have a default value in the macro for the empty state self. So when you want to use the defaults you only have to add the Link. I switched the order multiple times based on were and how I needed in in Fork, this order seems to be the most practical way in my eyes.

Reply:
I prefer the current order as well